docs/content/preview/migrate/yb-voyager/_index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: YugabyteDB Voyager
+headerTitle: YugabyteDB Voyager
+linkTitle: YugabyteDB Voyager
+description: Migrate to YugabyteDB with YugabyteDB Voyager.
+beta: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+image: /images/section_icons/develop/learn.png
+headcontent: Migrate to YugabyteDB with YugabyteDB Voyager.
+menu:
+  preview:
+    identifier: yb-voyager
+    parent: migrate
+    weight: 100
+---
+
+YugabyteDB Voyager is an open-source database migration engine provided by YugabyteDB. The engine manages the entire lifecycle of a database migration, including cluster preparation for data import, schema-migration and data-migration, using [yb-voyager](https://github.com/yugabyte/yb-voyager).
+
+Learn more about the [Migration workflow](../../migrate/yb-voyager/reference/) using YugabyteDB Voyager in the reference section.
+
+## Migration modes
+
+- Live migration : The source database can continue to change during the migration. After the full initial migration, yb-voyager continues replicating source database changes to the target database. The process runs continuously till you decide to switch over to the YugabyteDB database.
+
+- Offline migration: The source database should not change during the migration. The offline migration is considered complete when all the requested schema objects and data are migrated to the target database.
+
+{{< note title="Note" >}}
+yb-voyager supports only `offline` migration mode. The `live` migration mode is currently under development. For more details, refer to the [github issue](https://github.com/yugabyte/yb-voyager/issues/50).
+{{< /note >}}
+
+## Source databases
+
+YugabyteDB currently supports migrating schema and data from your existing RDBMS to a YugabyteDB cluster using  **YugabyteDB Voyager**. Using YugabyteDB Voyager, you can:
+
+- [Migrate from PostgreSQL](../yb-voyager/install-yb-voyager/#postgresql)
```

Comment:
@ddhodge , I understand the necessity for simplifying the link title. But the reason I retained the titles to [Migrate to PostgreSQL] and not [PostrgreSQL] even though it had repeated words was because, what if a reader is just taking a quick glance avoiding all the intro and other details ... Isin't the title [Migrate to PostgreSQL] more suggestive ?

Reply:
I don't find the links very helpful - they don't link to 'migrating', but to preparing your database for migration. I would get rid of the links, and add a reference below the list, something like "Before starting a migration using yb-voyager, you need to prepare your source database. For instructions on preparing your database for migration, refer to [Prepare the source database](../install-yb-voyager/#prepare-the-source-database).